src/Decompiler/Decompiler.cs

```diff
@@ -394,7 +394,7 @@ public void WriteGlobals(Program program, string filename, TextWriter w)
             w.WriteLine();
         }
     
-        public void WriteDecompiledTypes(Program program, string headerFilename, TextWriter w)
+        public void WriteDecompiledDecls(Program program, string headerFilename, TextWriter w)
```

Comment:
Consider using the name `WriteDecompiledDeclarations` here.

Reply:
Done in a1e0d60